ADOPTERS.md

```diff
@@ -0,0 +1,14 @@
+# Antrea Adopters
+
+<a href="http://glasnostic.com" border="0" target="_blank"><img alt="glasnostic.com" src="docs/assets/adopters/glasnostic-logo.png" height="50"></a>&nbsp; &nbsp; &nbsp;
+<a href="http://infrabuilder.com" border="0" target="_blank"><img alt="infrabuilder.com" src="docs/assets/adopters/infrabuilder-logo.png" height="50"></a>&nbsp; &nbsp; &nbsp;
+
+## Adding yourself as an Adopter
+
+It would be great to have your logo on our list of Antrea adopters! To add yourself you can simply drop your logo in PNG or SVG format to the [adopters](docs/assets/adopters) directory and submit a pull request with your change. Name the image file something that reflects your company (e.g., if your company is called Acme, name the image acme-logo.png). Check out the Antrea [Contributor Guide](CONTRIBUTING.md) on how to create a pull request. Alternatively, feel free to reach out via Slack or on Github to have our team add your logo.
```

Comment:
would be better to wrap this line at ~80 characters

Reply:
Done